src/vhost_user/message.rs

```diff
@@ -114,10 +114,30 @@ pub enum MasterReq {
     POSTCOPY_END = 30,
     /// Get a shared buffer from slave.
     GET_INFLIGHT_FD = 31,
-    /// Send the shared inflight buffer back to slave
+    /// Send the shared inflight buffer back to slave.
     SET_INFLIGHT_FD = 32,
+    /// Sets the GPU protocol socket file descriptor.
+    GPU_SET_SOCKET = 33,
+    /// Ask the vhost user backend to disable all rings and reset all internal
+    /// device state to the initial state.
+    RESET_DEVICE = 34,
+    /// Indicate that a buffer was added to the vring instead of signalling it
+    /// using the vring’s kick file descriptor.
+    VRING_KICK = 35,
+    /// Return a u64 payload containing the maximum number of memory slots.
+    GET_MAX_MEM_SLOTS = 36,
+    /// Update the memory tables by adding the region described.
+    ADD_MEM_REG = 37,
```

Comment:
How about renaming as ADD_MEM_REGION/REMOVE_MEM_REGION?

Reply:
Well I think this is better to stick to the vhost-user spec for this kind of naming. It makes the readability with the spec better, hence it helps maintaining the code.

Reply:
I agree, using the same naming as in vhost-user spec helps when you're "grepping" around.

Reply:
Yeh, the other thing I like to do if the name is not really "compatible" with the Rust way is I add a comment with the kernel/C specification name.